doc/wg/core/notes/core-notes-2022-11-18.md

```diff
@@ -0,0 +1,222 @@
+   about actions people take.
+ * Amit: Can you clarify that distinction?
+ * Phil: I think there is a difference between saying "Copyrights should be at
+   the top" and "you should not remove a copyright". The first is about what a
+   compliant document looks like (the artifact), the second is about the
+   actions that can be taken by an individual.
+ * Amit: What is the takeaway here?
+ * Phil: I am unsure whether this document should do the latter.
+ * Leon: I consider the latter as arguably more important.
+ * Johnathan: My concern with the latter is getting too close to giving legal
+   advice
+ * Phil: I think that is a good articulation of my concern. It is fair to say
+   that we do not expect developers not to remove copyrights, but that is
+   different from saying `SHOULD NOT`.
+ * Amit: I am personally swayed by that argument
+ * Leon: I think the current document is written using explicit RFC style
+   language to perform or not perform any actions. I am indifferent as to
+   whether we should be doing that.
+ * Phil (chat): SHOULD NOT: This phrase, or the phrase "NOT RECOMMENDED" mean
+   that there may exist valid reasons in particular circumstances when the
+   particular behavior is acceptable or even useful, but the full implications
+   should be understood and the case carefully weighed before implementing any
+   behavior described with this label.
+ * Leon: It is best for this document to establish some new legal framework and
+   instead just state best practices
+ * Amit: I agree, this should be a statement of our expectations, nothing more.
+ * Amit: OK, lets quickly go through the outstanding comments on the PR like
+   Hudson suggested
+ * Phil: Yeah I think hammering through these seems productive.
+ * Amit: Should we update from "exactly one" blank line after license/copyright
+   to "at least one"?
+ * consensus: yes
+ * Amit: Johnathan separated notices into two distinct groups, Tock project
+   added notices and 3rd party notices. Do we agree that we should limit our
+   formatting requirements and automated enforcement to the former group? I
+   believe we do?
+ * Leon: How are we going to enforce this in practice? Via an allowlist?
+ * Johnathan: The tool will look for a Tock project notice at the top of the
+   file in a particular format. If it does not find it it complains, if it does
+   find it, it stops looking.
+ * Johnathan: The document should be clear that these requirements only apply
+   to Tock project notices.
+ * Amit: I will make a note of that to add as a comment at the end
+ * Phil: So the resolution here is that the formatting expectations are about
+   the Tock project license headers/copyright notices.
+ * Leon: That is nice because it means this document does not cover copyright
+   enforcement
+ * Johnathan: My next comments says that if another project's license header we
+   should be willing to accept theirs rather than nearly duplicating it at the
```

Comment:
`if another project's license header we should` -> `if another project's license header is similar to ours we`